src/main/java/Gicha/repository/CharacterRepository.java

```diff
@@ -0,0 +1,6 @@
+package Gicha.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CharacterRepository extends JpaRepository<Character, Integer> {
```

Comment:
Jsi si jista datovym rozhrani v generice?

Reply:
?
nechapu otazku

Reply:
Ptam se, jestli jsi si jista timhle rozhranim. Co tohle dela, specificky, vysvetli mi, co v tomto pripade dela Integer

<Character, Integer>

Reply:
nedela nic, opravila jsem to na long (ID Type)
:')